features/works/work_edit_multiple.feature

```diff
@@ -28,22 +28,54 @@ Feature: Edit Multiple Works
       And I should see "Lovely"
 
   Scenario: I can edit multiple works at once
-    Given I am logged in as "author"
-      And I post the work "Glorious" with fandom "SGA"
-      And I post the work "Excellent" with fandom "Star Trek"
-      And I go to my works page
+  Given I am logged in as "author"
+    And I post the work "Glorious" with fandom "SGA"
+    And I post the work "Excellent" with fandom "Star Trek"
+    And I go to my works page
+  When I follow "Edit Works"
+  Then I should see "Edit Multiple Works"
+    And I should see "All"
+    And I should see "None"
+  When I select "Glorious" for editing
+    And I select "Excellent" for editing
+    And I press "Edit"
+  Then I should see "Your edits will be applied to all of the following works"
+    And I should see "Glorious"
+    And I should see "Excellent"
+  When I set the fandom to "Random"
+   And I press "Update All Works"
```

Comment:
I think this line needs to be indented one more space.

Reply:
Fixed